pkg/servicescm/servicescm_test.go

```diff
@@ -238,6 +238,60 @@ func TestValidateDependencies(t *testing.T) {
 			},
 			expectedErr: true,
 		},
+		{
+			name: "Cyclical dependency structure",
+			input: []Service{
+				{
+					Name:         "new-bootstrap-service",
+					Command:      "C:\\new-service",
+					Dependencies: []string{"test-controller-service"},
+					Bootstrap:    false,
+					Priority:     0,
```

Comment:
It may be easier to read if you use the priority value to fill all the fields. For example:

```
					Name:         "service-0",
					Command:      "C:\\service-0",
					Dependencies: []string{"service-1"},
					Bootstrap:    false,
					Priority:     0,
```